plugins/osdn/multitenant/multitenant.go

```diff
@@ -31,6 +31,9 @@ func Master(osClient *osclient.Client, kClient *kclient.Client, clusterNetwork s
 	if err != nil {
 		glog.Fatalf("SDN initialization failed: %v", err)
 	}
+	// TODO: Get admin namespaces from master node config
```

Comment:
There has to be a programmable way of specifying admin namespaces. I would think its feature incomplete if we do not allow the choice.